src/layers.c

```diff
@@ -20,6 +19,26 @@
 #include "labwc.h"
 #include "node.h"
 
+static void
+arrange_one_layer(struct output *output, const struct wlr_box *full_area,
+		struct wlr_box *usable_area, struct wlr_scene_tree *tree,
+		bool exclusive)
+{
+	struct wlr_scene_node *node;
+	wl_list_for_each(node, &tree->children, link) {
+		/* ignore nodes in the process of being destroyed */
+		if (!node || !node->data) {
```

Comment:
I don't think `!node` can happen here.

The `!node->data` shouldn't happen either but better safe than sorry.

Reply:
Agree, it shouldn't happen. I can't force any errors, so have removed. Was probably being over-cautious.